SWLOR.Game.Server/Perk/MartialArts/CircleKick.cs

```diff
@@ -84,6 +85,11 @@ private void ApplyFeatChanges(NWPlayer oPC, NWItem unequippingItem)
             CustomItemType offType = offHand.CustomItemType;
             bool receivesFeat = true;
 
+            
+
+            //_nwnxCreature.AddFeat(oPC, NWScript.FEAT_CIRCLE_KICK);
```

Comment:
Looks like you left some funny debugging code here. :) Try to patch this up please